x-pack/plugins/cases/public/client/api/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpStart } from 'kibana/public';
+import { CasesByAlertId, getCasesFromAlertsUrl } from '../../../common/api';
+
+export const createClientAPI = ({ http }: { http: HttpStart }) => {
```

Comment:
This is the only new code of this PR

Reply:
Can we get http from `useHttp` instead of getting it directly?

Reply:
We cannot use the `useHttp` because this is not a React component but a method.  Same as the Core team export to us the [`getCurrentUser`](https://github.com/elastic/kibana/blob/main/x-pack%2Fplugins%2Fsecurity%2Fpublic%2Fauthentication%2Fauthentication_service.ts#L58)

Reply:
I totally missed that part. Ignore my comment please :)